### ISSUE TITLE
Embedded refresh undefined

### ISSUE THREAD

**Issue by tiansin** [fabiospampinato/vscode-todo-plus#93]
I click the refresh button, a undefined.
![1](https://user-images.githubusercontent.com/4232281/43935256-acde62ce-9c85-11e8-9520-f1b0f85ee6c1.png)

**Comment by fabiospampinato** [comment]
I think you'll have to update your `todo.embedded.regex` setting.

v3 and v4 featured breaking changes.

> An actual regex will be generated from the value of the todo.embedded.regex setting. It uses 2 capturing groups, the first one captures the type of the todo (TODO, FIXME etc.) and the second one captures an optional description (TODO: description).

**Comment by tiansin** [comment]
[front.zip](https://github.com/fabiospampinato/vscode-todo-plus/files/2276389/front.zip)
![gif](https://user-images.githubusercontent.com/4232281/43935899-a31dcf9c-9c88-11e8-9b4d-ac10031f5fc3.gif)

I recorded a animation and test directory and upload the animation.

**Comment by fabiospampinato** [comment]
I can't reproduce it. 

I see the relative paths of files, instead of their absolute path, as it was meant to be:

<img width="294" alt="screen shot 2018-08-10 at 04 38 40" src="https://user-images.githubusercontent.com/1812093/43936070-4787a3fa-9c57-11e8-9492-45b77cdb473c.png">

Maybe there's an issue with those chinese (?) characters or something, can you send me the full path to that "front" directory in your computer?

**Comment by tiansin** [comment]
@fabiospampinato 
Yes, the directory contains Chinese, which is the directory I used when testing.
`E:\桌面\front`

**Comment by fabysdev** [comment]
@fabiospampinato I also have the "undefined".

For me it happens in an workspace with 2 directories:
![image](https://user-images.githubusercontent.com/2088490/43941631-2b56d598-9c75-11e8-86a5-e6c078b97ad8.png)


The first time i go to the tab it is not grouped by root.
![image](https://user-images.githubusercontent.com/2088490/43941635-2f676c24-9c75-11e8-9ac5-cecd5880639d.png)


If I press the refresh button the root groups appear + an "undefined" which contains all todos without being grouped by the root.
![image](https://user-images.githubusercontent.com/2088490/43941643-34c3a30e-9c75-11e8-9b08-7cbfcc9f7751.png)

I changed the following embedded.views settings to:
```
  "todo.embedded.view.expanded": false,
  "todo.embedded.view.wholeLine": true,
```

I am on windows.

**Comment by fabiospampinato** [comment]
It might be a Windows issue then, I'll look into that, thanks.

**Comment by fabiospampinato** [comment]
I can reproduce this when using Windows, fix coming soon 👍

**Comment by fabiospampinato** [comment]
Should be fixed in v4.0.5, thanks for reporting the error!
